standards/core/lightpush.md

```diff
@@ -0,0 +1,88 @@
+---
+title: WAKU-LIGHTPUSH
+name: Waku Light Push
+editor: Zoltan Nagy <zoltan@status.im> 
+contributors: 
+  - Hanno Cornelius <hanno@status.im>
+  - Daniel Kaiser <danielkaiser@status.im>
+  - Oskar Thorén <oskarth@titanproxy.com>
+---
+
+previous version: `/vac/waku/lightpush/2.0.0-beta1` [19/WAKU2-LIGHTPUSH](https://rfc.vac.dev/waku/standards/core/19/lightpush)
+
+---
+**Protocol identifier**: `/vac/waku/lightpush/3.0.0`
+
+## Motivation and Goals
+
+Light nodes with short connection windows and limited bandwidth wish to push messages to other nodes in the Waku network to request message services.
+A common use case is to request that the service node publish the message to an `11/WAKU2-RELAY` pubsub-topic.
+Additionally, there is sometimes a need for confirmation that a message has been received "by the network"
+(here, at least one node).
+
+`WAKU-LIGHTPUSH` is a request/response protocol for this.
+
+## Payloads
+
+```protobuf
+syntax = "proto3";
+
+message LightPushRequest {
+    string request_id = 1;
+    uin32 request_type = 10;  // 10 Reserved for future `request_type`. Currently, RELAY is the only available service.
+    optional string pubsub_topic = 20;
+    WakuMessage message = 21;
+}
+
+message LightPushResponse {
+    string request_id = 1;
+    uint32 status_code = 10; // non zero in case of failure, see appendix
+    optional string status_desc = 11;
+    optional uint32 relay_peer_count = 12; // number of peers, message is successfully relayed to 
+}
+```
+
+### Message Relaying
+
+Nodes that respond to `LightPushRequest` MUST either relay the encapsulated message via [11/WAKU2-RELAY](https://rfc.vac.dev/waku/standards/core/11/relay) protocol on the specified `pubsub_topic`. Depending on the network configuration, the user may not need to provide `pubsub_topic` ([WAKU2-RELAY-SHARDING](https://github.com/waku-org/specs/blob/master/standards/core/relay-sharding.md)).
+If the node is unable to do so for some reason, they SHOULD return an error code in `LightPushResponse`.
+Once the relay is successful, the `relay_peer_count` will indicate the number of peers that the node has managed to relay the message to. It's important to note that this number may vary depending on the node subscriptions and support for the requested pubsub_topic. The client can use this information to either consider the relay as successful or take further action, such as switching to a lightpush service peer with better connectivity.
+The field `relay_peer_count` may not present or has the value zero in case of error or in other future use cases, where no relay is involved.
```

Comment:
```suggestion
The field `relay_peer_count` may not be present or has the value zero in case of error or in other future use cases, where no relay is involved.
```